torchtitan/datasets/hf_datasets.py

```diff
@@ -30,6 +33,11 @@
 _supported_datasets = {
     "c4_test": "test/assets/c4_test",
     "c4": "allenai/c4",
+    "train_mini": "test/assets/train_mini"
```

Comment:
Let's give it a name like chemlactica_train_mini

Reply:
OK